### ISSUE TITLE
Use doctrine metadata for entity guessing

### ISSUE THREAD

**Issue by spolischook** [nelmio/alice#661]
As far as I know now used typehinting for entity guessing and find by id.
But doctrine metadata is more precisely now what type of entity should be seted

**Comment by theofidry** [comment]
Good point, not sure if it's not what's already done but will double check :)
